### PR TITLE
Completed - take home lab/javascript

### PR DESCRIPTION
**Pull request by Ryaong792**

### CODE REVIEW

**Comment by Ryaong792** [comment]
COMPELTEED -  if you want a 5 finger count I would give it a 1 finger for sure.  
need to review my basic and get my foundation down.  Stuck to it and found the indexOf and Math.abs.  1am seeing + and - all over the place.

**Comment by Ryaong792** [comment]
removed bad comments

File: lib/mbta.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
+    return (Math.abs(endSubway - startSubway))
+  } else if (startLine !== endLine) {
+  let startIntersection = mbtaSubway[startLine].indexOf('Park Street')
+  let endIntersection = mbtaSubway[endLine].indexOf('Park Street')
+  return (Math.abs((startSubway + startIntersection) - (endSubway + endIntersection)))
+  }
 }
 
+// note: return Math.abs (end - start) needs to be (Math.abs(end-start))
+// Need to really review with flash cards.  also calulations was wrong needed
```
Comment (benjimelito) [inline lib/mbta.js:59]: Don't worry about flash cards or memorization, you'll get more comfortable with this stuff the more you use it. Until then, making sure your Google Fu is on point will help you the most

File: lib/mbta.js [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
+  endLine, endStation){
+  let startSubway = mbtaSubway[startLine].indexOf(startStation)
+  let endSubway = mbtaSubway[endLine].indexOf(endStation)
+  if (startLine === endLine) {
+    return (Math.abs(endSubway - startSubway))
+  } else if (startLine !== endLine) {
+  let startIntersection = mbtaSubway[startLine].indexOf('Park Street')
+  let endIntersection = mbtaSubway[endLine].indexOf('Park Street')
+  return (Math.abs((startSubway + startIntersection) - (endSubway + endIntersection)))
+  }
```
Comment (benjimelito) [inline lib/mbta.js:55]: Glad to see you got the `Math.abs` and `indexOf` keys down for this practice

**Comment by benjimelito** [comment]
Don't worry about frustration, it's bound to happen. Lean on your fellow developers and the consultants when the going gets tough 👍

**Pull request closed without merging**